storage/src/mmr/verification.rs

```diff
@@ -10,8 +10,6 @@ use commonware_cryptography::{Digest, Hasher as CHasher};
 /// their peak hashes from the elements within the range. Both segments are ordered by decreasing
 /// height.
 #[derive(Clone, Debug, Eq)]
-/// A Proof contains the information necessary for proving the inclusion of an element, or some
```

Comment:
While you're here did you also want to drop the reference to `Proof` above in an appropriately short description?

Reply:
I'll leave this to you to adjust in your PR.

Reply:
already done!